### ISSUE TITLE
Test Failure: com.ibm.ws.logging.fat.TimeBasedLogRolloverTest.testMaxFileSizePriorityRolling

### ISSUE THREAD

**Issue by fmhwong** [OpenLiberty/open-liberty#21026]
RTC 290553

```
testMaxFileSizePriorityRolling:java.lang.Throwable: 2022-05-04-15:43:15:306 There were 2 errors:
  java.net.ConnectException(Can't assign requested address (connect failed))
  java.lang.RuntimeException(Server stop failed with RC 34.
Stdout:

Stopping server com.ibm.ws.logging.timedrolloverxml.
The command stop failed because of a communication error with the server.
Server com.ibm.ws.logging.timedrolloverxml stop failed. Check server logs for details.

Stderr:
)
at componenttest.custom.junit.runner.FATRunner.newThrowableWithTimeStamp(FATRunner.java:295)
at componenttest.custom.junit.runner.FATRunner.access$700(FATRunner.java:67)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:267)
at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:320)
at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:173)
  java.net.ConnectException(Can't assign requested address (connect failed))
  java.lang.RuntimeException(Server stop failed with RC 34.
Stdout:

Stopping server com.ibm.ws.logging.timedrolloverxml.
The command stop failed because of a communication error with the server.
Server com.ibm.ws.logging.timedrolloverxml stop failed. Check server logs for details.

Stderr:
)
at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:199)
```

**Comment by fmhwong** [comment]
@ellen-lau Any update on this?

**Comment by ellen-lau** [comment]
fixed by: #21733
